test/task.js

```diff
@@ -53,7 +60,7 @@ describe('task', function() {
   });
 
   it('should register a named function by string name', function(done) {
-    taker.task('test1', noop);
+    expect(taker.task('test1', noop)).to.be.undefined();
```

Comment:
Because I wanted to check that `task` returned nothing when working as a setter.
Is it better to change back to original?